zilencer/models.py

```diff
@@ -47,6 +47,10 @@ class Plan(models.Model):
     # The two possible values for nickname
     CLOUD_MONTHLY = 'monthly'
     CLOUD_ANNUAL = 'annual'
+    # Subscription amount in U.S Dollars
```

Comment:
Let's hold off on this change; I'm not sure exactly how we'll want to do it, and we can hold off until we need it.